data/helpers.d/filesystem

```diff
@@ -80,3 +80,31 @@ properly with chmod/chown." >&2
     chmod 755 $TMP_DIR
     echo $TMP_DIR
 }
+
+# Remove a file or a directory securely
+#
+# usage: ynh_secure_remove path_to_remove
+# | arg: path_to_remove - File or directory to remove
+ynh_secure_remove () {
+	path_to_remove=$1
```

Comment:
I think, at a time or another we will have some critics about this limitation. Because, someone will want to remove several directories at one time. Or even the content of a directory with a regex.
But, however, it can be a real problem if a wildcard it badly uses.

So, I think using a such limitation can be the better way to keep the point of this helper.